docs/ReleaseNotes.md

```diff
@@ -1,6 +1,15 @@
 Release Notes for Version 14
 ============================
 
+Build 014
+-------
+Published as version 14.8.1
+
+New Features:
```

Comment:
New features require a minor version update, so v14.9.0. But, I think this is not a new feature. It's more of an update/fix to existing features. I would put it in the bug fixes section instead